### ISSUE TITLE
More granular font introspection

### ISSUE THREAD

**Issue by cfilipov** [nalexn/ViewInspector#70]
Currently, it seems the only way to inspect font properties is by an actual instance of `Font` as demonstrated in the guide.

```swift
extension InspectableView {
    
    func find(textWithFont font: Font) throws -> InspectableView<ViewType.Text> {
        return try find(ViewType.Text.self, where: {
            try $0.attributes().font() == font
        })
    }
}

let text = try sut.find(textWithFont: .headline)
```

Imagine the design team you are working with has specified that all text in the app should use the Gotham typeface instead of the system font. One might use ViewInspector to automatically highlight all text that is not Gotham so that it stands out during testing. In order to do this today I would need to pass in multiple `Font` instances for each possible variation (bold, italic, each font size etc...).

Is it possible to expose the font family name, point size or other properties that you would find on UIFont? Is there at least some string representation that can be pulled out (so that one could do a substring compare for the font name).

**Comment by nalexn** [comment]
@cfilipov I'll add an extension to the `Font` that allows for extracting the name, size, and TextStyle

That's what's stored inside a custom `Font` struct:

```
let font = Font.custom("Gotham", size: 34)
po Inspector.print(font) as AnyObject
Font
  provider: FontBox<NamedProvider>
    base: NamedProvider
      name: String = Gotham
      size: CGFloat = 34.0
      textStyle: Optional<TextStyle>
        some: TextStyle = body
```

**Comment by nalexn** [comment]
Released with v0.6.1
